README.md

```diff
@@ -209,12 +210,15 @@ Status will be null if the :id user is either the current user or they have no r
 
 ### Routes
 
-| Type | Route | Description |
-| ---: | ----- | ----------- |
-| `DELETE` | `/users` | Delete the current user by id |
-| `GET` | `/users/:id` | Get user object by id |
-| `PUT` | `/users/:id` | Modify the user object by id |
-| `GET` | `/users/search/:name` | Get list of users matching the name string |
+|     Type | Route                 | Description                                |
+| -------: | --------------------- | ------------------------------------------ |
+|    `GET` | `/users`              | Get the current user's object by id        |
+|    `GET` | `/users/:id`          | Get user object by id                      |
+|    `PUT` | `/users/:id`          | Modify the user object by id               |
+| `DELETE` | `/users/:id`          | Delete the user user by id. O              |
+|    `GET` | `/users/search/:name` | Get list of users matching the name string |
+
+As a quick note, the delete route will only work if the id passed in is the same as the current user's id. otherwise, a `403` status will be returned.
```

Comment:
You should put this comment underneath the ## `DELETE /users/:id` section in the ### Fetch Format